### DISCUSSION TITLE
【求助】卡牌店老板生意惨淡，希望有绳匠帮忙支招

### DISCUSSION THREAD

**Discussion by 132641515Zzy** [share121/inter-knot#98] [其他]
![image](https://github.com/user-attachments/assets/730c9f48-b745-4c76-af1f-177954eb71a0)
在艾利都开了一家卡牌店，但是苦于没有人来，朋友帮忙求来一批卡牌提高纯度，现有帅气英雄、劈柴红狼、弱小人鱼等等，希望有砖家帮忙鉴定强度

**Comment by myc-bm** [comment]
听上去都是很强的卡，我这还有一些卡片可以提供，包括巨龙天使，零攻妹妹，零攻大姐姐。

**Reply by 132641515Zzy** [reply]
一看就是内行

**Comment by qiyana6** [comment]
高价出售：世界仅有一张的纯金无敌白龙，详情请前往五分街114514号

**Reply by 132641515Zzy** [reply]
盗版吧，全世界仅有的四张一张在我这，另外三张在海马小子那，你怕不是古鲁斯吧

**Comment by JJC-555** [comment]
这个委托我接不了

**Reply by 132641515Zzy** [reply]
脚本去----，不对
